requirements-dev.txt

```diff
@@ -9,4 +9,5 @@ aiohttp==2.2.5
 botocore==1.6.4
 multidict==3.1.3
 wrapt==1.10.11
-dill==0.2.7.1
\ No newline at end of file
+dill==0.2.7.1
```

Comment:
I think we should fix this to follow this model: https://github.com/spulec/moto/blob/master/requirements-dev.txt  where it pulls from requirements.txt .. in this case instead will look more like: https://github.com/spulec/moto/blob/master/requirements-dev.txt

Reply:
👍

Reply:
@thehesiod could you please elaborate? I am bit confused what is problem here?